rules/cloud/azure/azure_ad_azurehound_discovery.yml

```
@@ -0,0 +1,23 @@
+title: AzureHound activity detected
+id: 35b781cc-1a08-4a5a-80af-42fd7c315c6b
+status: experimental
+description: AzureHound is a BloodHound data collector for Microsoft Azure. This detects AzureHound activity via the default User-Agent that is used during its operation after successful authentication.
+references:
+    - https://github.com/BloodHoundAD/AzureHound
+author: Janantha Marasinghe
+date: 2022/11/27
+tags:
+    - attack.t1087.004
+    - attack.t1526
+    - attack.discovery
+logsource:
+    product: azure
+    service: signinlogs
+detection:
+    selection_ua:
```

```suggestion
    selection:
```
as condition use selection